src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -264,6 +264,8 @@ static String getResource(String key) {
             {"R_clientKeyPropertyDescription", "Private key file path for client certificate authentication feature."},
             {"R_clientKeyPasswordPropertyDescription",
                     "Password for private key if the private key is password protected."},
+            {"R_sendTemporalDataTypesAsStringForBulkCopyPropertyDescription",
+                    "Flag to indicate whether the driver will send temporal datatypes as String value to the server for bulk copy."},
```

Comment:
suggest that we be consistent in how we describe our other sendBlahAsBlah properties, ie "Determines whether the driver will send..."

Reply:
I don't think my new addition here is inconsistent because other descriptive properties don't follow the "Determines whether..." starters either. I think the suggestion is good though.